### PR TITLE
Remove blacklisting of headers in aws auth

### PR DESCRIPTION
**Pull request by harryi3t**

### CODE REVIEW

**Comment by shamasis** [comment]
@czardoz any thought?

**Comment by czardoz** [comment]
@shamasis @harryi3t I had a chat with you guys offline, so I think this could be merged. Unless something else came up.

**Comment by kamalaknn** [comment]
@harryi3t can you add a changelog entry for this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Postman Runtime Changelog
 
+### Unreleased
```
Comment (kamalaknn) [inline CHANGELOG.md]: ####

**Pull request merged**